tests/e2e/logging/test_openshift-logging.py

```diff
@@ -2,22 +2,27 @@
 This file contains the testcases for openshift-logging
 """
 
-import pytest
 import logging
+import pytest
```

Comment:
pytest is third party library so if we are following pep8 conventions for imports you should have one empty line above. See here: https://www.python.org/dev/peps/pep-0008/#imports (You should put a blank line between each group of imports.)